docs/source/reference/exchange.rst

```diff
@@ -0,0 +1,44 @@
+:orphan:
```

Comment:
I am thinking make it "Pack management migration" or like this; and reserve StackStorm exchange for a permanent doc entry about the exchange, how it works, etc. 

some of the stuff here will live, and some only live for 1 version.